Makefile

```diff
@@ -0,0 +1,272 @@
+PROJECT_NAME=devopsconsole-operator
+PACKAGE_NAME:=github.com/fabric8-services/$(PROJECT_NAME)
```

Comment:
this should be `github.com/redhat-developer`

Reply:
Fixed befe3ef.